src/Serializers/Orleans.Serialization.Protobuf/Orleans.Serialization.Protobuf.csproj

```diff
@@ -1,20 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <PackageId>Microsoft.Orleans.OrleansGoogleUtils</PackageId>
-    <Title>Microsoft Orleans Google Utilities</Title>
-    <Description>Library of utility types for Google of Microsoft Orleans.</Description>
-    <PackageTags>$(PackageTags) ProtoBuf</PackageTags>
-    <TargetFrameworks>$(DefaultTargetFrameworks)</TargetFrameworks>
-  </PropertyGroup>
 
   <PropertyGroup>
-    <AssemblyName>Orleans.Serialization.Protobuf</AssemblyName>
-    <RootNamespace>OrleansGoogleUtils</RootNamespace>
+    <PackageId>Microsoft.Orleans.Serialization.Protobuf</PackageId>
```

Comment:
We forgot to delete these projects...

Reply:
Ok, is the current location ok, or should I move the Orleans.Serialization.Protobuf project to the /src dir instead of /src/Serializers?

LMK if I should delete the whole section: https://github.com/dotnet/orleans/tree/main/src/Serializers